docs/en/04_Changelogs/4.9.0.md

```diff
@@ -1,6 +1,27 @@
 # 4.9.0 (Unreleased)
 
 
-## New features
+## Features and enhancements {#features-and-enhancements}
+
+### Image lazy-loading set by default
+
+HTML `<img>` tags now have the `loading="lazy"` attribute added by default on images that are:
+- Inserted via the WYSIWYG editor
+- Added to a silverstripe template as an image object that redners as HTML e.g. `<div>$MyImage</div>`
+
+Other `<img>` tags such as those hardcoded in Silverstripe CMS templates and HTML returned in API responses remain unchanged.
+
+Individual images can have lazy-loading disabled via the following methods:
+- WYSIWYG images - Click on the image, then click the "Edit image" icon.  Modify the value in form control in the image placement form
+- Image objects in templates - suffix with `.LazyLoad(false)` in the SilverStripe CMS template (*to be confirmed*)
```

Comment:
```suggestion
- Image objects in templates - suffix with `.LazyLoad(0)` in the SilverStripe CMS template
```

SilverStripe template syntax does not handle booleans yet, but this is something we could consider cherry-picking. https://github.com/silverstripe/silverstripe-assets/pull/457#discussion_r662835648

Reply:
looks like there's still some discussion here, it would be preferable to do LazyLoad(false), even if that's actually just passing a string "false".  I'll just leave as is for now, we can easily update these docs later